src/components/CaseManagerUI/paras.tsx

```diff
@@ -0,0 +1,60 @@
+import { trpc } from "client/lib/trpc";
+import Link from "next/link";
+import React from "react";
+
```

Comment:
Re parent directory naming: in general, prefer snake_case for directories, e.g. "case_manager". Suffixes like "UI" can be dropped as  this is already in the components directory. 

The actual components themselves can PascalCased as recommended by AirBnb's style guide: https://github.com/airbnb/javascript/tree/master/react#naming